.travis.yml

```diff
@@ -41,3 +41,8 @@ addons:
 
 script:
   - make ci-generate ci-build ci-test ci-deploy ci-docker
+  # Testing to see if go-ci-test catches the same failures as ci-test. If I
+  # understand how travis works, this means that go-ci-test will run even if
+  # ci-test fails, so we can see if they're catching the same
+  # things. Likewise, if go-ci-test fails, we can compare that to ci-test.
+  - make go-ci-test
```

Comment:
Do we want to do this inline immediately after `ci-test` on the line above? The only reason I mention this is because as it is if `go-ci-test` fails, then the build will have failed, but it will still have produced docker images.

Reply:
> Do we want to do this inline immediately after `ci-test` on the line above? The only reason I mention this is because as it is if `go-ci-test` fails, then the build will have failed, but it will still have produced docker images.

Yes, I thought about that, but since we consider ci-test the "official" source of a failure, that seemed to be the preferred outcome.

Also in that case, if ci-test fails, then go-ci-test isn't run, so we won't know if it "matched".

It's all a matter of "well, what do you want to do?", ya know? This is where I landed. 

I really don't think we'll have any trouble with one failing when the other doesn't though, do you?

Reply:
I just think it may be confusing where the build fails, but it gets deployed anyhow (assuming the feature branch was deployed to an environment).

If the team is okay with it, then I'm good.